171014_Django-vue.js/django-polls-app/README.md

```diff
@@ -0,0 +1,12 @@
+# Django Polls App
+
+## Co to je
+
+Jedná se o ukázkovou aplikaci zpracovanou podle tutoriálu https://docs.djangoproject.com/en/1.11/intro/tutorial01/. Databáze je přiložena, takže se o ni nemusíme nijak starat :-) V aplikaci můžeme odpovědět na dvě jednoduché otázky a udržovat si statistiku odpovědí. V administrátorském rozhraní je možné taky otázky přidávat nebo upravovat.
+
+## Jak zprovoznit
+0. Nainstaluj si Python, vytvoř virtuální prostředí a aktivuj ho ([návod](http://naucse.python.cz/2017/pyladies-praha-podzim-cznic/beginners/install/)), nainstaluj si git ([návod](http://naucse.python.cz/2017/pyladies-praha-podzim-cznic/git/install/))
+1. Nainstaluj si Django; většinou by mělo zadat do příkazové řádky ```pip install Django``` (pokud by to nešlo, je tu i [komplexnější návod](https://docs.djangoproject.com/en/1.11/topics/install/))
+2. Naklonuj (```git clone git@github.com:pyvec/pyworking-materials.git```, případně [stáhni]((https://github.com/pyvec/pyworking-materials/tree/master/171014_Django-vue.js)) si tento repozitář.
```

Comment:
Tady bych spíš dal https url, ať to jde stáhnout i lidem, co nemají nakonfigurovaný svůj SSH klíč v Githubu.

Reply:
Dobrý tip, já jsem zvyklý používat ssh, protože mám pocit, že https zlobí s 2FA, ale tady to asi většina lidí používat nebude, takže to nevadí.